packages/core/upload/admin/src/components/AssetTableList/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { BaseCheckbox } from '@strapi/design-system/BaseCheckbox';
+import { Table, Th, Thead, Tr } from '@strapi/design-system/Table';
+import { Typography } from '@strapi/design-system/Typography';
+import { VisuallyHidden } from '@strapi/design-system/VisuallyHidden';
+
+import { getTrad } from '../../utils';
+import { AssetDefinition } from '../../constants';
+import { tableHeaders } from './utils/tableHeaders';
+import { TableRows } from './TableRows';
+
+export const AssetTableList = ({ assets, onEditAsset, onSelectAsset, selectedAssets }) => {
+  const { formatMessage } = useIntl();
+
+  const headers = tableHeaders.map((header) => ({
+    ...header,
+    metadatas: {
+      ...header.metadatas,
+      label: formatMessage(header.metadatas.label),
+    },
+  }));
+
+  return (
+    <Table colCount={tableHeaders.length} rowCount={assets.length}>
+      <Thead>
+        <Tr>
+          <Th>
+            {/* TODO: Replace ML BaseCheckbox select all with this one when adding folders to Table */}
+            <BaseCheckbox
+              aria-label={formatMessage({
+                id: getTrad('bulk.select.label'),
+                defaultMessage: 'Select all folders & assets',
+              })}
+              onValueChange={() => {}}
```

Comment:
How come we're passing this?

Reply:
it's a placeholder for this checkbox that will replace the current bulk select one, `Table` UI is broken without a placeholder for this column 😬 - I can remove it until next PR if you prefer

Reply:
all good if you're planning on replacing in the next PR :)